osu.Game/Screens/SelectV2/BeatmapLeaderboardScore.cs

```diff
@@ -144,6 +144,7 @@ private void load()
                 {
                     background = new Box
                     {
+                        Alpha = 0.4f,
```

Comment:
Not sure how much you're going to care about it but this affects daily challenge screen too.

| before | after |
| :-: | :-: |
| ![Screenshot 2025-05-29 at 13 10 23](https://github.com/user-attachments/assets/c72c2294-36e9-45a7-94ea-93fa52cd4e66) | ![Screenshot 2025-05-29 at 13 10 55](https://github.com/user-attachments/assets/a4535384-0add-4ffc-89b4-356c6a51bff8) |

The usage there always sucked but maybe now it sucks a bit more. Dunno. It's still sorta legible, for better or worse.

Reply:
Seems okay still, I think.